Add section to the developer page and link to it in the r frnr0f8 (#1489)
bring/developer-site: content/api/tracking/index.md

weight: 31
 
important:
- type: warn
  title: Rate limiting unauthenticated users - 20.03.2023
  message: | 

>    The Tracking API will start rate limiting unauthenticated users on 20 March 2023 and require authentication from May 1, 2023. 

Reviewer: May 1st? That's already been, no?

Author: Yes, the date is wrong. Just need a new date, forgot about that

     To meet these expectations, we must know which customers and partners are using our APIs and how they are using them.

>      To further develop the value proposition of our Tracking API, the solution becomes authenticated from 01.05.2023.

Reviewer: ```suggestion
     To further develop the value proposition of our Tracking API, the solution will require authentication from 01.05.2023.
```

>    To meet these expectations, we must know which customers and partners are using our APIs and how they are using them. 

Reviewer 2: Should we provide some text saying that personal data etc will not be used, or describe how we are using the data? People might have questions about what _"we must know which customers and partners are using our APIs, and how they are using them"_ means, as we are saying we will identify you and look at your usage to identify how we can improve our services.

Author: Might be a good idea

Author: @richas2 What do you think?

>      Starting 20th of March 2023, the Tracking API will start rate limiting unauthenticated users.

Reviewer 3: One suggestion:
- It’s generally a good idea to start off with what’s happening, not the reason, suggesting rewriting the last two paragraphs and moving them to the start: `The Tracking API will start rate limiting unauthenticated users on 20 March 2023 and require authentication from May 1 2023. Our customers […]`
- Notce the date format is the same as the existing message below. Further ref.: https://www.mybring.com/design-system/guidelines/numbers-units/ The date format in the title is fine.
- This also fixes `the solution the solution` which was repeated.

weight: 31
 
important:
  - type: warn

>     title: Rate limiting our unauthenticated users - 20.03.2023

Reviewer: ```suggestion
    title: Rate limiting unauthenticated users - 20.03.2023
```

    message: |
      Starting 20 March 2023, the Tracking API will start rate limiting our unauthenticated users. This will happen gradually for the next two months.

>       Ending with Tracking API only being available for authenticated users, to read more about this look at the authentication section

Reviewer: ```suggestion
      After these two months have passed, the Tracking API will only be available for authenticated users. To read more about how to authenticate, see the authentication section
```

Reviewer: Also, is it possible to link to the authentication section?

      To make authenticated Tracking API requests: You will need an API key from Mybring. Steps for getting a key and description of headers can be found on the general API [Getting Started / Authentication](/api/#authentication) page.

>       The Tracking API will stop being open to anyone and start requiring user to be authenticated. We have started rate limiting our unauthenticated, this will result in

Reviewer: ```suggestion
      The Tracking API will stop being open to anyone and start requiring user to be authenticated. We have already started rate limiting unauthenticated users, which will result in
```
